src/cobalt.rs

```diff
@@ -158,7 +154,7 @@ pub fn build(config: &Config) -> Result<()> {
                 }
 
                 try!(fs::copy(entry.path(), &dest.join(relative))
-                         .map_err(|_| format!("Could not copy {:?}", entry.path())));
+                    .map_err(|_| format!("Could not copy {:?}", entry.path())));
```

Comment:
I know you didn't change this in this PR but should we not tell the user what the error was. Like permissions?